### ISSUE TITLE
BUG: Cannot see full list of groups in Safari on login

### ISSUE THREAD

**Issue by vukilelanga** [Coko-Foundation/Kotahi#414]
When opening the root page of Kotahi on Safari, the dropdown list of groups is obstructed by Safari's password autofill. The user must first press `esc` before being able to see the full list of users can see the items at the top of the list.

<img width="413" height="462" alt="Image" src="https://github.com/user-attachments/assets/66770e48-47ac-42f8-afa9-46641558e3e7" />
